book-1-orientation/chapters/QUEST.md

```diff
@@ -0,0 +1,287 @@
+# An Adventurer's Quest
+
+## A C# Exercise
+
+This exercise we will don the brightly colored robe and shiny hat of an adventurer and undergo a series of challenges to complete a quest.
+
+### Phase 1
+
+In this phase we'll get going with some starter code. We'll add to this initial code in future phases.
+
+1. Create a new console application called _"Quest"_.
+1. Initialize a git repo for the new app, add a `.gitignore` file and commit the code.
+1. Create a new repo in GitHub and push your code to it.
+1. Add two new C# source code files. _(Do not remove the `Program.cs` file)_
+    * `Adventurer.cs`
+    * `Challenge.cs`
+1. Update the `*.cs` files to contain the code shown below.
+
+    > `Program.cs`
+
+    ```cs
+    using System;
+    using System.Collections.Generic;
+
+    // Every class in the program is defined within the "Quest" namespace
+    // Classes within the same namespace and refer to one another without a "using" statement
+    namespace Quest
+    {
+        class Program
+        {
+            static void Main(string[] args)
+            {
+                // Create a few challenges for our Adventurer's quest
+                // The "Challenge" Constructor takes three arguments
+                //   the text of the challenge
+                //   a correct answer
+                //   a number of awesome points to gain or lose depending on the success of the challenge
+                Challenge twoPlustwo = new Challenge("2 + 2?", 4, 10);
+                Challenge theAnswer = new Challenge(
+                    "What's the answer to life, the universe and everything?", 42, 25);
+                Challenge whatSecond = new Challenge(
+                    "What is the current second?", DateTime.Now.Second, 50);
```

Comment:
Every time I read this I expect the answer to be the seconds to midnight on the doomsday clock rather than the literal time. Not a suggestion, just a comment.